InventoryComponents/InventoryConsumablesComponent.gd

```diff
@@ -33,13 +33,16 @@ func remove_consumable_at_place(pos : int) -> void:
 	_held_consumables[pos] = null
 	held_consumable_update.emit(null, pos)
 
+## calls the on_consume of the item in the slot and then removes it
 func use_consumable_at_place(pos : int) -> void:
 	if(_held_consumables[pos] == null):
 		return
 	
 	_held_consumables[pos].on_consume()
 	remove_consumable_at_place(pos)
 
+## fills the _held_consumables array with empty slots or removes slots if there are too many
```

Comment:
forgot a [br] tag at end of line. You can look at the doc inside the Godot doc searcher to see how it looks.
Unless you meant for it to all be on a single line in the doc.